MinnPost/object-sync-for-salesforce
# 154 custom post object save custom fields

## What does this PR do?

This release fixes the issue with custom fields on custom objects not being sent on the first save, causing required Salesforce fields to fail.

## How do I test this PR?

- create a custom object with custom fields
- map those custom fields to required fields in Salesforce
- they should successfully be added on first save now.
